test/bdd/agent_controller_steps.go

```diff
@@ -0,0 +1,211 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package bdd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/DATA-DOG/godog"
+
+	"github.com/hyperledger/aries-framework-go/pkg/client/didexchange"
+	"github.com/hyperledger/aries-framework-go/pkg/restapi/operation/didexchange/models"
+)
+
+const (
+	operationID           = "/connections"
+	createInvitationPath  = operationID + "/create-invitation"
+	receiveInvtiationPath = operationID + "/receive-invitation"
+	acceptInvitationPath  = operationID + "/{id}/accept-invitation"
+	connections           = operationID
+	connectionsByID       = operationID + "/{id}"
+	acceptExchangeRequest = operationID + "/{id}/accept-request"
+	removeConnection      = operationID + "/{id}/remove"
+
+	AliceAgentHost       = "${ALICE_AGENT_HOST}"
```

Comment:
Same as https://github.com/hyperledger/aries-framework-go/pull/574#discussion_r338580158.

Reply:
What if we want to run a 3rd agent (eg Faber) ? Do we need to update the code like below ?

```
	FaberAgentHost         = "${FABER_AGENT_HOST}"
	FaberAgentPort         = "${FABER_AGENT_PORT}"
	FaberAgentController   = "${FABER_CONTROLLER_URL}"
	FaberAgentWebhook      = "${FABER_WEBHOOK_URL}" 
```

Reply:
Yes, because we need to start docker containers before launching tests.